app/views/layouts/application.html.erb

```diff
@@ -10,8 +10,9 @@
   <%= stylesheet_link_tag    'application', media: 'all' %>
 </head>
 <body>
+  <%= render 'layouts/navbar' %>
   <p class="notice"><%= notice %></p>
-  <p class="alert"><%= alert %></p>
+  <p class=r"alert"><%= alert %></p>
```

Comment:
@chamyndu 
Please correct the following stuff: 

- put the navabr in the shared 

- delete the "r" in the p tag line 15